github/is-member-of/action.yml

```diff
@@ -24,13 +24,20 @@ runs:
     - id: gh-api-is-member
       name: Check if user is member of the given org
       run: |
-        if gh api \
+        # required because 'gh api' will return non-zero for 404s to indicate lack of membership
+        set +e
+        # gh api will return a 404 with status to indicate lack of membership
+        result=$(gh api \
           -H "Accept: application/vnd.github+json" \
-          /orgs/${{ env.GH_ORG }}/members/${{ env.GH_USER }} ; then
+          /orgs/${{ env.GH_ORG }}/members/${{ env.GH_USER }} 2>/dev/null | jq -r '.status // "member"' )
+
+        if [[ '404' != "${result}" ]]; then
```

Comment:
I'm not sure whether we need some `set -o pipefail` so if the pipe in `L32` failed then `result=false`. What do you think?